src/connectors/panoptic/panoptic.controllers.ts

```diff
@@ -609,14 +707,15 @@ export async function forceExercise(
     timestamp: startTimestamp,
     nonce: tx.nonce,
     txHash: tx.hash,
+    other: tx
```

Comment:
Let's rename this to `tx`

Reply:
Done: 63437c76ef72876e68cfa57e508ef824e90919b4